Automattic/simplenote-android
# Update Note List

### Fix
Update the note list interface for better performance and readability.  These changes include updating the note list item layout and floating action button icon.  The navigation bar style will be included in a separate pull request.  See the screenshots below for illustration.

![update_note_list_expanded_light](https://user-images.githubusercontent.com/3827611/63825843-838e0980-c919-11e9-9d41-dd8ac74bde85.png)

![update_note_list_expanded_dark](https://user-images.githubusercontent.com/3827611/63825846-85f06380-c919-11e9-8050-bf17896fbc1d.png)

![update_note_list_condensed_light](https://user-images.githubusercontent.com/3827611/63825859-96a0d980-c919-11e9-811a-e99997f35c41.png)

![update_note_list_condensed_dark](https://user-images.githubusercontent.com/3827611/63825863-99033380-c919-11e9-90a9-6419482f783c.png)

### Test
Follow the steps below with the ***Condensed note list*** setting enabled and disabled with both ***Light*** and ***Dark*** themes applied to verify the layouts shown in the screenshots above.
1. Go to note list.
2. Notice updated note list item layout.
  a. Notice list dividers are hidden between items.
  b. Notice published icon is shown for published notes.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 42f2345 with:
> Updated note list interface for better performance and readability
